IN:WinevtLogs - added documentation for "ignore_missing_channels" (#1568)
fluent/fluent-bit-docs: pipeline/inputs/windows-event-log-winevtlog.md

| DB | Set the path to save the read offsets. \(optional\) |  |
| String\_Inserts | Whether to include StringInserts in output records. \(optional\) | True  |
| Render\_Event\_As\_XML | Whether to render system part of event as XML string or not. \(optional\) | False  |

> | Ignore\_Missing\_Channels | Whether to ignore event channels not present in the event log and keep running with channels that could be subscribed to. \(optional\) | False  |

Reviewer: ```suggestion
| Ignore\_Missing\_Channels | Whether to ignore event channels not present in the event log, and continue running with subscribed channels. \(optional\) | False  |
```
